examples/relative_free_energy.py

```diff
@@ -279,6 +93,7 @@ def read_from_args():
     parser.add_argument("--mol_b_name", type=str, help="name of the end molecule", required=True)
     parser.add_argument("--forcefield", type=str, help="location of the ligand forcefield", required=True)
     parser.add_argument("--protein", type=str, help="PDB of the protein complex", required=True)
```

Comment:
Is this meant to be specific to hif2a or general? args.protein seems unused here

Reply:
should be in general, willfix